runtime/doc/usr_05.txt

```diff
@@ -120,7 +120,7 @@ This switches on three very clever mechanisms:
 				*restore-cursor* *last-position-jump*  >
     augroup RestoreCursor
       autocmd!
-      autocmd BufRead * autocmd FileType <buffer> ++once
+      autocmd BufRead * ++once
```

Comment:
Actually now that I review the change I'm not sure anymore. I thought the previous version was messed up by accident. Now I noticed the `++once`. Not sure what was the idea here.

In any case: The initial version did not work for me.

Reply:
It seems that `BufRead` is too late to attach to `FileType`. `BufReadPre` works fine, though. Commit updated.